shop.py

```diff
@@ -0,0 +1,75 @@
+splash = '''
+
+                    ███████╗██╗  ██╗ ██████╗ ██████╗     ██████╗  ██████╗ ████████╗
+                    ██╔════╝██║  ██║██╔═══██╗██╔══██╗    ██╔══██╗██╔═══██╗╚══██╔══╝
+                    ███████╗███████║██║   ██║██████╔╝    ██████╔╝██║   ██║   ██║
+                    ╚════██║██╔══██║██║   ██║██╔═══╝     ██╔══██╗██║   ██║   ██║
+                    ███████║██║  ██║╚██████╔╝██║         ██████╔╝╚██████╔╝   ██║
+                    ╚══════╝╚═╝  ╚═╝ ╚═════╝ ╚═╝         ╚═════╝  ╚═════╝    ╚═╝
+
```

Comment:
Fancy. But before I forget to mention, the Werewolf Bot ASCII art didn't show up too well on the terminal, soooo....

Reply:
> Use a better terminal

Reply:
I'll say `git gud` in the terminal, see if that helps.

Reply:
The ASCII is no longer printed to console